java/maven/pom.xml

```diff
@@ -274,6 +288,7 @@
       <plugin>
         <groupId>org.cyclonedx</groupId>
         <artifactId>cyclonedx-maven-plugin</artifactId>
+        <version>2.7.11</version>
```

Comment:
@lidavidm just curious, I am comparing this PR agains: https://github.com/apache/arrow/pull/41309. 

And We the following dependency in the main
```pom
<plugin>
  <groupId>org.cyclonedx</groupId>
  <artifactId>cyclonedx-maven-plugin</artifactId>
  <version>2.8.0</version>
</plugin>
```

And we had the following in the said PR. 

```pom
<plugin>
  <groupId>org.cyclonedx</groupId>
  <artifactId>cyclonedx-maven-plugin</artifactId>
  <version>2.7.11</version>
</plugin>
```

And in here we have `2.7.11`, any specific reason we picked 2.7.11? not `2.8.0`?